packages/springboard/webpack.config.ts

```diff
@@ -1,9 +1,19 @@
 import { getCommonConfig, HtmlWebpackPlugin } from '@packages/web-config/webpack.config.base'
 import path from 'path'
-import webpack from 'webpack'
+import webpack, { Plugin } from 'webpack'
 import { VueLoaderPlugin } from 'vue-loader'
+import WindiCSS from 'windicss-webpack-plugin'
 
 const common = getCommonConfig()
+
+// existing webpack config relating to styles
```

Comment:
Dunno what's going on here, just got it working for now. Probably need to update quite a few webpack deps at some point... seems mini-css-plugin is conflicting with windicss somehow.